database/migrations/blocktx/postgres/000005_add_inserted_at.up.sql

```diff
@@ -1,5 +1,12 @@
+ALTER TABLE blocks
+ADD COLUMN inserted_at_num INTEGER DEFAULT TO_NUMBER(TO_CHAR((NOW()) AT TIME ZONE 'UTC', 'yyyymmddhh24'), '9999999999') NOT NULL;
```

Comment:
Why do we need this column?

Reply:
Will this default value correctly convert to hours?

Reply:
In my test it converted to hours. When I do this for example:
`SELECT TO_NUMBER(TO_CHAR((NOW()) AT TIME ZONE 'UTC', 'yyyymmddhh24'), '9999999999');`
I get
`2023121112`